### PR TITLE
Add Rscript dockerfiles

### PR DESCRIPTION
**Pull request by elingg**
Please take a look at your earliest convenience, @ssk2. Thx!

### CODE REVIEW

File: rscript/r-script-curl-aws [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:14.10
```
Comment (ssk2) [inline rscript/r-script-curl-aws]: If you set the `FROM` to `FROM mesosphere/rscript-curl` then you can avoid re-installing all the packages it has installed. This'll be MUCH quicker.

**Comment by ssk2** [comment]
Looks good apart from the FROM.

File: rscript/r-script-curl-aws (hunk 1 above)
Comment (elingg) [inline rscript/r-script-curl-aws]: great point @ssk, will do that!

**Comment by elingg** [comment]
Made the update. Any final objections, @ssk2?

**Comment by ssk2** [comment]
Looks good - make sure you link to the Dockerfiles in the description of the Docker images when published.

**Comment by elingg** [comment]
@ssk2, yep. Will do that now.

**Pull request merged**